I'm sorry, but an uncaught exception occurred.

While loading <'Image' u'cg_blood.png'>:
  File "game/script.rpy", line 108, in script call
    call check from _call_check
  File "game/script.rpy", line 82, in script call
    call game3 from _call_game3
  File "game/script.rpy", line 1919, in script
    show cg_blood2 at Position(xpos=renpy.random.randint(100, 600), ypos=renpy.random.randint(0, 1000)) with vpunch
IOError: Couldn't find file 'cg_blood.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 108, in script call
    call check from _call_check
  File "game/script.rpy", line 82, in script call
    call game3 from _call_game3
  File "game/script.rpy", line 1919, in script
    show cg_blood2 at Position(xpos=renpy.random.randint(100, 600), ypos=renpy.random.randint(0, 1000)) with vpunch
  File "D:\renpy\renpy-6.99.13-sdk\renpy\ast.py", line 1356, in execute
    renpy.exports.with_statement(trans, paired)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\exports.py", line 1481, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\core.py", line 2227, in do_with
    clear=clear)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\core.py", line 2672, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\core.py", line 3158, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\core.py", line 2075, in draw_screen
    renpy.config.screen_height,
  File "render.pyx", line 487, in renpy.display.render.render_screen
  File "render.pyx", line 235, in renpy.display.render.render
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\layout.py", line 722, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\motion.py", line 179, in render
    child = render(self.child, width, height, st, at)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\layout.py", line 722, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\layout.py", line 722, in render
    surf = render(child, width, height, cst, cat)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "accelerator.pyx", line 110, in renpy.display.accelerator.transform_render
  File "render.pyx", line 235, in renpy.display.render.render
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\layout.py", line 336, in render
    surf = render(self.child, width, height, st, at)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\image.py", line 500, in render
    return wrap_render(self.target, width, height, st, at)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\image.py", line 306, in wrap_render
    rend = render(child, w, h, st, at)
  File "render.pyx", line 147, in renpy.display.render.render
  File "render.pyx", line 235, in renpy.display.render.render
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\im.py", line 580, in render
    return cache.get(self, render=True)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\im.py", line 266, in get
    surf = image.load()
  File "D:\renpy\renpy-6.99.13-sdk\renpy\display\im.py", line 625, in load
    surf = renpy.display.pgrender.load_image(renpy.loader.load(self.filename), self.filename)
  File "D:\renpy\renpy-6.99.13-sdk\renpy\loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'cg_blood.png'.

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
버드나무 꽃 만개할 그날까지 1.1
Wed Mar 27 09:04:13 2019
